app/views/layouts/_drawer.mobile.haml

```diff
@@ -35,6 +35,13 @@
           = person_image_tag(current_user, size: :thumb_small)
       %li= link_to t("_contacts"), contacts_path
       %li= link_to t("layouts.header.settings"), edit_user_path
+      - if current_user.moderator? && !current_user.admin?
```

Comment:
Can you do that the same way [as it is done on desktop](https://github.com/diaspora/diaspora/blob/526ebcfa13a799cae6bfbcc822c958ebecfa4d2b/app/assets/templates/header_tpl.jst.hbs#L102-L106)? So put it as `- elsif current_user.moderator?` below the `- if current_user.admin?` block, then you don't need `&& !current_user.admin?`